src/test/java/LoginTests.java

```diff
@@ -71,7 +69,7 @@ private String getErrorMessage () {
 
      public WebElement findElement (String cssLocator) {
 
-         return driver.findElement ( By.cssSelector (cssLocator ) );
+         return wait.until ( ExpectedConditions.elementToBeClickable ( By.cssSelector ( cssLocator ) ) );
      }
```

Comment:
It would be better move this method to Base class

Reply:
SetUpPageTest? what is that? I  don't have it

Reply:
Sorry, I ment "SetUpPage"